docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -558,6 +558,8 @@ _Properties_
 
 ### getLastInsertedBlockClientId
```

Comment:
If I'm not wrong this didn't land in WordPress at all, we should be able to remove it

Reply:
Ah so it doesn't count for Gutenberg Plugin just for WP Core? So as long as it didn't land in a WP release we're all good to remove?